apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -71,12 +69,6 @@ defmodule SiteWeb.ScheduleController.LineController do
         route_patterns: conn.assigns.route_patterns,
         stop_tree: conn.assigns.stop_tree,
         alerts: conn.assigns.alerts,
-        line_diagram:
-          update_route_stop_data(
-            conn.assigns.all_stops,
```

Comment:
**question**: does this mean we can remove `plug(SiteWeb.ScheduleController.AllStops)` from this controller?

Reply:
I don't think so, or I guess more accurately I don't know enough to be able to say that. The `AllStops` plug is a generic plug that is used in many places and actually gets all the stops for a route, just like it sounds like it would. Confusingly, just for this controller in [line.ex](https://github.com/mbta/dotcom/pull/1441/files#diff-7091bf01cdd3f25d37f063b7a084da7e63b948517b44497a6024e5de5262720aL147) we were overwriting that value with a completely different, unrelated data that had nothing to do with the more general `all_stops`. So I guess maybe? But if so, it never should have been there in the first place, which makes me uncertain because I don't know whether there would be unintended consequences.